.github/workflows/ci.yml

```diff
@@ -22,9 +22,9 @@ jobs:
           sudo systemctl status clamav-daemon.service
           head /etc/clamav/clamd.conf
       - name: pip install
-        run: pip install -r requirements.txt -c constraints_plone52.txt
```

Comment:
constraints_plone52.txt is no longer needed, delete it?

Reply:
Yeah. We have to clean up a bit.